src/sentry/scim/endpoints/teams.py

```diff
@@ -235,3 +226,16 @@ def put(self, request, organization, team):
         # override parent's put since we don't have puts
         # in SCIM Team routes
         return self.http_method_not_allowed(request)
+
+    def _get_member_id_for_remove_op(self, operation):
+        if "value" in operation:
+            # azure sends member ids in this format under the key 'value'
+            return operation["value"][0]["value"]
+
+        try:
+            # grab the filter out of the brackets of the string that looks
+            # like so: members[value eq "123124"]
+            filter_path = re.search(r"\[(.*?)\]", operation["path"]).groups()[0]
+            return parse_filter_conditions(filter_path)[0]
+        except ValueError:
```

Comment:
Can this be narrowed? It's unclear if we expect the `ValueError` from the regex search or from `parse_filter_conditions`.

Regarding the regex search, it might be worth checking whether `search` returns `None` before calling `.groups()` in order to get a clearer error in case the path doesn't have any brackets.

Reply:
👍🏼 will make these changes after other PR is merged